client/src/components/Related/ProductCarousel.jsx

```diff
@@ -0,0 +1,93 @@
+/* eslint-disable guard-for-in */
+/* eslint-disable no-restricted-syntax */
+/* eslint-disable no-console */
+import React from 'react';
+import axios from 'axios';
+import card from './card.css';
+
+class ProductCarousel extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      productInfo: [],
+    };
+  }
+
+  componentDidMount() {
+    this.getInfo();
+  }
+
+  getInfo() {
+    // axios.get('/products/${current product id}/related')
+    axios.get('/products/48432/related')
+      .then((res) => {
+        // console.log('list of related ids', res.data);
+        const relatedIds = res.data;
+        const ids = [];
+        for (let i = 0; i < relatedIds.length; i++) {
+          const relatedId = relatedIds[i];
+          ids[i] = { relatedId };
+        }
+        this.setState({
+          productInfo: ids,
+        });
+        const { productInfo } = this.state;
+        for (const id in productInfo) { // [{relatedId: 48433}, {…}, {…}, {…}]
+          const { relatedId } = productInfo[id]; // [48433, 48434, 48439, 48438]
+          axios.get(`/products/${relatedId}`)
+            .then((res) => {
+              const relatedCategories = res.data;
```

Comment:
@r-jeng It looks like "relatedCategories" isn't being used in this file, so you can remove this line, or replace all the "res.data" with "relatedCategories"

Reply:
Removed this line and the following console.log